Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf

```diff
@@ -6,6 +6,245 @@
 #pragma ModuleName=MIES_OOD
 #endif
 
+/// @brief returns the threshold level for ood region detection from a single column stimset
+/// @param[in] stimset 1d wave containing stimset data
+/// @return threshold level defining signal above baseline
+static Function OOD_GetThresholdLevel(stimSet)
+	WAVE stimSet
+
+	variable minVal = WaveMin(stimset)
+	return minVal + (WaveMax(stimset) - minVal) * 0.10
+End
+
+/// @brief retrieves regions with signal from a 1D data wave, used for stimsets
+/// @param[in] stimset 1D wave containing stimset data
+/// @param[in] prePoints pre delay in points the regions get expanded at the rising edge
```

Comment:
@MichaelHuth minor comment: how about:
 `prePoints oodDAQ pre delay in points.....` 
and 
 `postPoints oodDAQ post delay in points.....`